apps/frontend/src/features/battle/components/result/RankListSection.tsx

```
@@ -68,20 +73,27 @@ export const RankListSection = ({
                   </div>
                   <span css={userNameStyle}>{ranking.displayName}</span>
                 </div>
-                <div css={scoreValueStyle}>{ranking.score}</div>
+                <div
+                  css={scoreValueStyle}
+                  aria-label={`${ranking.displayName} 점수: ${ranking.score}`}
+                >
+                  {ranking.score}
+                </div>
```

Biome (2.3.13)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)